prime-router/settings/staging/0166-flexion-staging-results-handling.yml

```diff
@@ -0,0 +1,135 @@
+        operation: MERGE
+        numberPerDay: 1440 # Every minute
+        initialTime: 00:00
+      translation: !<FHIR>
+        type: "FHIR"
+        schemaName: "metadata/fhir_transforms/receivers/fhir-transform-sample"
+        useBatching: false
+      transport: !<REST>
+        type: "REST"
+        authType: "two-legged"
+        authTokenUrl: "https://cdcti-stg-api.azurewebsites.net/v1/auth/token"
+        reportUrl: "https://cdcti-stg-api.azurewebsites.net/v1/etor/orders"
+        # Uncomment below for local Trusted-Intermediary server
+        # authTokenUrl: "http://host.docker.internal:8080/v1/auth/token"
+        # reportUrl: "http://host.docker.internal:8080/v1/etor/orders"
+        tlsKeystore: null
+        parameters:
+          scope: "report-stream"
+          client_assertion: "client_secret"
+        headers:
+          Content-Type: "text/fhir+ndjson"
+          RecordId: "header.reportFile.reportId"
+          senderLabName: "CDC PRIME REPORTSTREAM" #name-of-the-lab-the-sending-user-belongs-to
+          sourceLabName: "CDC PRIME REPORTSTREAM" #name-of-the-lab-that-originally-sent-this-hl7-message-to-OSDH
+        authHeaders:
+          Content-Type: "application/x-www-form-urlencoded"
+          Host: "cdcti-stg-api.azurewebsites.net"
+    # RS => PHL: translates OML_O21 from FHIR to HL7 and uploads the message to the SFTP folder
+    - name: simulated-lab
+      externalName: Simulated State Public Health Lab ETOR Receiver - Receiving OML_O21 HL7 orders
+      organizationName: flexion
+      topic: etor-ti
+      customerStatus: active
+      jurisdictionalFilter:
+        - Bundle.entry.resource.ofType(MessageHeader).event.code = 'O21' # OML_O21
+      qualityFilter:
+        - true
+      timing:
+        operation: MERGE
+        numberPerDay: 1440 # Every minute
+        initialTime: 00:00
+      translation: !<HL7>
+        type: "HL7"
+        schemaName: "metadata/hl7_mapping/receivers/Flexion/TILabOrder"
+        useTestProcessingMode: false
+        useBatchHeaders: false
+      transport:
+        type: SFTP
+        host: sftp
+        port: 22
+        filePath: ./upload
+        credentialName: DEFAULT-SFTP
+    # Results Receivers
+    # RS => TI: translates ORU_R01 from HL7 to FHIR and uploads the message to the SFTP folder
+    - name: etor-service-receiver-results
+      externalName: ETOR Service Receiver Channel from ReportStream for results - Receiving ORU_R01 FHIR results
+      organizationName: flexion
+      topic: etor-ti
+      customerStatus: active
+      jurisdictionalFilter:
+        - Bundle.entry.resource.ofType(MessageHeader).event.code = 'R01' # ORU_R01
+      qualityFilter:
+        - true
+      timing:
+        operation: MERGE
+        numberPerDay: 1440 # Every minute
+        initialTime: 00:00
+      translation: !<FHIR>
+        type: "FHIR"
+        schemaName: "metadata/fhir_transforms/receivers/fhir-transform-sample"
+        useBatching: false
+      transport:
```

Comment:
For staging this should be:

      host: "172.17.6.20"
      port: "22"
      filePath: "./upload"
      credentialName: null
      type: "SFTP"